pkg/generate/app/cmd/newapp_test.go

```diff
@@ -53,6 +53,17 @@ func TestValidate(t *testing.T) {
 			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
 			parms:               map[string]string{},
 		},
+		"envfile": {
+			cfg: AppConfig{
+				GenerationInputs: GenerationInputs{
+					EnvironmentFile: "/tmp/envfile1",
+				},
+			},
+			componentValues:     []string{},
+			sourceRepoLocations: []string{},
+			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
+			parms:               map[string]string{},
+		},
```

Comment:
line 109, is having a call to cfg.validate(). Isn't that enough. Am I missing anything here? I printed the output of Environment in c.Environment and its having values from file that we read